app/src/main/kotlin/com/mapzen/erasermap/presenter/RoutePresenter.kt

```diff
@@ -6,13 +6,17 @@ import com.mapzen.valhalla.Instruction
 import com.mapzen.valhalla.Route
 
 public interface RoutePresenter {
+    companion object {
+        public val GESTURE_MIN_DELTA = 0.0001f
```

Comment:
Good point. I'm not certain but [MotionEvent.html#getHistoricalX(int, int)](https://developer.android.com/reference/android/view/MotionEvent.html#getHistoricalX(int, int)) does return a float so I'm guessing its within the realm of posibility.